clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureComputeClient.groovy

```diff
@@ -45,6 +48,16 @@ public class AzureComputeClient extends AzureBaseClient {
     this.client = this.initialize(credentials)
   }
 
+
+  @Lazy
+  private VirtualMachineImagesOperations vmImageOps = {client.getVirtualMachineImagesOperations()}()
+
+  @Lazy
+  private VirtualMachineScaleSetsOperations scaleSetOps = {client.getVirtualMachineScaleSetsOperations()}()
+
+  @Lazy
+  private VirtualMachineScaleSetVMsOperations scaleSetVMOps = {client.getVirtualMachineScaleSetVMsOperations()}()
+
```

Comment:
What's the lifetime expectation for these objects? Is it reasonable to expect that they will never "expire"?